Client/Application/AppDelegate.swift

```diff
@@ -257,9 +257,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             log.warning("Cannot handle \(components.scheme) URL scheme")
             return false
         }
+        
+        // Extract optional FxA deep-linking options
+        let fxaQuery = url.getQuery()
+        let fxaParams: FxALaunchParams
+        fxaParams = FxALaunchParams(view: fxaQuery["fxa"], email: fxaQuery["email"], access_code: fxaQuery["access_code"])
```

Comment:
Hey @st3fan, It doesn't appear so. I opened the following deep links in safari and got the following results. These seem to be correct.

- `firefox://?fxa=signin&email=test@email.com`

Launches FxiOS, opens the FxA Content View Controller and prefills email.

- `firefox://?fxa=signin`

Launches FxiOS, opens the FxA Content View Controller

- `firefox://`

Launches FxiOS